test/init-tables.py

```diff
@@ -0,0 +1,37 @@
+"""
+init-tables.py
+
+A Python script that adds a few tables to Deephaven.
+
+@author Jake Mulford
+@copyright Deephaven Data Labs
+"""
+from pydeephaven import Session
+
+import time
+
+session = None
+
+#Simple retry loop in case the server tries to launch before Deephaven is ready
+count = 0
+max_count = 5
```

Comment:
Is 10 sec long enough?

Reply:
I can bump it to 25 to be extra safe, typically it's up by this point but can't hurt